modules/standards/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@phenyl/standards",
+  "version": "1.0.0",
+  "description": "",
+  "repository": "phenyl-js/phenyl",
+  "license": "Apache License 2.0",
+  "author": "Shin Suzuki <shinout310@gmail.com> ,Nicholas Lee <nicholaslee119@gmail.com>",
+  "files": [
+    "src",
+    "dist",
```

Comment:
`dist`, `jsnext.js`, `decls` can be removed from `files`. Also, add `lib` to files